base/stats.go

```diff
@@ -461,6 +461,8 @@ type DatabaseStats struct {
 	DocWritesXattrBytes *SgwIntStat `json:"doc_writes_xattr_bytes"`
 	// Highest sequence number seen on the caching DCP feed.
 	HighSeqFeed *SgwIntStat `json:"high_seq_feed"`
+	// Number of bytes written over public interface for REST api
```

Comment:
This looks like it's just a subset of CBG-2929 (doesn't include request bytes over REST, and doesn't include blip).  If it makes sense to split into separate PRs, creating subtickets under CBG-2929 seems like a good idea to make sure nothing gets missed.

Reply:
These are in https://github.com/couchbase/sync_gateway/commit/a31be806b47b07cb9e3d7fdfaaaa8ec024d2f0a8